docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md

```diff
@@ -1,44 +1,44 @@
-
-
 FAQ How do I use the platform logging facility?
 ===============================================
 
-The Eclipse runtime plug-in provides a simple set of APIs for logging exceptions, warnings, or other information useful in debugging or servicing a deployed Eclipse product. The intent of the log is to record information that can be used later to diagnose problems in the field. Because this information is not directed at users, you do not need to worry about translating messages or simplifying explanations into a form that users will understand. The idea is that when things go wrong in the field, your users can send you the log file to help you figure out what happened.
+The Eclipse ILog class provides a simple set of APIs for logging exceptions, warnings, or other information useful in debugging or servicing a deployed Eclipse product. 
+The intent of the log is to record information that can be used later to diagnose problems in the field. 
+Because this information is not directed at users, you do not need to worry about translating messages or simplifying explanations into a form that users will understand. 
+The idea is that when things go wrong in the field, your users can send you the log file to help you figure out what happened.
 
 Each plug-in has its own log associated with it, but all logged information eventually makes its way into the platform log file (see the getLogFileLocation method on Platform).
 
-You can write any kind of IStatus object to the log file, including a MultiStatus if you have hierarchies of information to display. If you create your own subclass of the utility class Status, you can override the getMessage method to return extra information to be displayed in the log file. Many plug-ins add utility classes for writing messages and errors to the log:
-
-      import org.eclipse.core.runtime.Status;
+      import org.eclipse.core.runtime.ILog;
       ...
       public class MyLogger {
       ...
       private static final Bundle BUNDLE = FrameworkUtil.getBundle(MyLogger.class);
```

Comment:
Is this line still necessary? 
At least while viewing this on the phone I can't see a reference.